iac/variables.tf

```diff
@@ -10,4 +10,16 @@ variable "github_client_secret" {
   nullable    = false
   description = "GitHub OAuth App Client Secret"
   sensitive   = true
-}
\ No newline at end of file
+}
+
```

Comment:
Extracting configuration values like `aws_region` and `domain_name` into variables makes the Terraform code reusable and easier to maintain. It separates configuration from logic.